pkg/settings/setting.go

```diff
@@ -130,6 +130,7 @@ var (
 	SystemFeatureChartRefreshSeconds    = NewSetting("system-feature-chart-refresh-seconds", "21600")
 	ClusterAgentDefaultAffinity         = NewSetting("cluster-agent-default-affinity", ClusterAgentAffinity)
 	FleetAgentDefaultAffinity           = NewSetting("fleet-agent-default-affinity", FleetAgentAffinity)
+	MaxUIPluginFileByteSize             = NewSetting("max-ui-plugin-file-byte-size", strconv.Itoa(20<<20)) // Max file size in bytes for ui plugins
```

Comment:
For clarity it would be nice to have the default value created as a `const` with a comment that tells how many MBs that the value represent. 

```golang
// Max file size in bytes for ui plugins
const defaultMaxUIPluginFileSizeInBytes = 20 * 1024 * 1024 // 20MB
```

Which is then used here:
```suggestion
	MaxUIPluginFileByteSize             = NewSetting("max-ui-plugin-file-byte-size", defaultMaxUIPluginFileSizeInBytes)
```

Reply:
Done